README.md

```diff
@@ -14,17 +14,31 @@ and [oemoflex](https://github.com/rl-institut/oemoflex).
 Oemof-B3 is currently under heavy development, which means that first full scenario runs will be
 available in the coming months.
 
+## Installation
+
+Currently, Oemof-B3 needs python 3.7 (newer versions may be supported, but installation can take very long).
+
+In order to install oemof-B3 locally, following steps must be made: 
+- git-clone oemof-B3 into local folder: `git clone https://github.com/rl-institut/oemof-B3.git`
+- enter folder
+- create virtual environment using conda: `conda env create environment.yml`
+- activate environment: `conda activate oemof_b3`
```

Comment:
```suggestion
- activate environment: `conda activate oemof-B3`
```

Should we call it `oemof-B3` for consistency? Would then have to be changed accordingly in `environment.yml`.